src/styles/index.scss

```diff
@@ -1,3 +1,58 @@
+@import './utils/variables';
+@import 'block/header';
+@import 'block/pruduct-card';
```

Comment:
There is a typo in the imported file name 'pruduct-card'. It should be 'product-card' as per the naming convention and to avoid import errors.